packages/react/src/components/organization-switcher/organization-switcher.test.tsx

```diff
@@ -62,49 +58,196 @@ describe.skip("OrganizationSwitcher", () => {
 
-  test.skip("should display the current organization as the selected value", () => {
-    expect(false).toBeTruthy()
-  })
+    const root = container.querySelector(".sid-organization-switcher")
+    const dropdown = container.querySelector(".sid-organization-switcher > .sid-dropdown")
+    const triggerDisabledState = dropdown?.querySelector<HTMLButtonElement>(".sid-dropdown__trigger")?.disabled
+    const label = dropdown?.querySelector(".sid-dropdown__trigger__label")?.innerHTML
 
-  test.skip("should display a menu on click", () => {
-    expect(false).toBeTruthy()
+    expect(root).toBeInTheDocument()
+    expect(dropdown).toBeInTheDocument()
+    expect(triggerDisabledState).toBe(false)
+    expect(label).toBe(TEXT["org.switcher.label"])
   })
 
-  test.skip("should render all organizations in menu", () => {
-    expect(false).toBeTruthy()
-  })
+  test("should display the current organization as the selected value", async () => {
+    const org = createTestOrganization()
+    const user = createTestUser({ oid: org.id })
+    const spy = vi.fn(async () => [org])
+
+    user.getOrganizations = spy
+    
+    const { container } = render(
+      <TestSlashIDProvider
+        user={user}
+      >
+        <OrganizationSwitcher />
+      </TestSlashIDProvider>
+    );
+    
+    await waitFor(() => expect(screen.queryByText("-")).not.toBeInTheDocument())
+
+    const currentValue = container.querySelector(".sid-organization-switcher > .sid-dropdown .sid-dropdown__trigger__input span")?.innerHTML
 
-  test.skip("should only render organizations which satisfy predicate in menu", () => {
-    expect(false).toBeTruthy()
+    expect(currentValue).toBe(org.org_name)
   })
 
-  test.skip("should render org.switcher.label in dropdown input", () => {
-    expect(false).toBeTruthy()
+  test("should display a menu on click", async () => {
+    const org = createTestOrganization()
+    const user = createTestUser({ oid: org.id })
+    const anotherOrg = createTestOrganization()
+    const spy = vi.fn(async () => [org, anotherOrg])
+    const event = userEvent.setup();
+
+    user.getOrganizations = spy
+    
+    const { container } = render(
+      <TestSlashIDProvider
+        user={user}
+      >
+        <OrganizationSwitcher />
+      </TestSlashIDProvider>
+    );
+    
+    await waitFor(() => expect(screen.queryByText("-")).not.toBeInTheDocument())
+    await waitFor(() => expect(container.querySelector(".sid-organization-switcher > .sid-dropdown .sid-dropdown__trigger")).toBeTruthy())
+
+    const trigger = container.querySelector(".sid-organization-switcher > .sid-dropdown .sid-dropdown__trigger")!!
+    
+    ;(window as any).PointerEvent = class PointerEvent extends Event {};
+    window.HTMLElement.prototype.scrollIntoView = vi.fn();
+    window.HTMLElement.prototype.hasPointerCapture = vi.fn();
+    window.HTMLElement.prototype.releasePointerCapture = vi.fn();
```

Comment:
This is nasty but I don't know what else to do about it:
https://github.com/testing-library/user-event/discussions/1087

Reply:
It is what it is - maybe leave a comment with the source?